packages/core/src/recon.ts

```diff
@@ -169,11 +169,10 @@ export class ReconApi extends Observable<ReconEventFeedResponse> implements IRec
     }
     try {
       const headers = { 'Content-Type': 'application/json' }
-      const body = { ...(controller && { controller }) }
-      await this.#sendRequest(this.#url + `/ceramic/interests/model/${model.toString()}`, {
+      const query = controller ? `controller=${controller}` : ''
+      await this.#sendRequest(this.#url + `/ceramic/interests/model/${model.toString()}?${query}`, {
```

Comment:
huh, should this be considered an issue with the API that C1 is exposing?  This is a POST request, why is it using URL query parameters instead of taking its args in the POST body?  Maybe the thing to do is change it in C1 instead of here?

CC @dav1do

Reply:
ah, yeah. that's actually the old one.. we have a newer one `POST /interests` that [behaves](https://github.com/ceramicnetwork/rust-ceramic/blob/89788719e3cffa649a60fb3166e1349537842b9c/api/ceramic.yaml#L232) like that

Reply:
Query params are not exclusive to get requests. I don't have a strong opinion either way just updated the client to match the server.

Reply:
@dav1do Thanks for the link to the new endpoint. I changed the code to use the new endpoint and manually tested. Working well...